### DISCUSSION TITLE
Desktop Client App

### DISCUSSION THREAD

**Discussion by Gond0r** [requarks/wiki#3796] [Help / Questions] [answered]
Hello, Thanks for the great project.
Is there a desktop client app for WikiJS to access my server using it instead of using the browser?

**Comment by NGPixel** [comment, marked as answer]
In most browsers, you can create an app from any website.  e.g.:

Chrome: Menu > More tools... > Create shortcut... (select Open as new window)
Edge: Menu > Apps > Install this site as an app

**Comment by Gond0r** [comment]
I meant to use something other than a browser based client...

**Reply by NGPixel** [reply]
The whole app is a website... which requires a browser. I'm not sure what a native desktop client would accomplish over a standard web app?

Many popular apps are embedded browser nowadays (e.g. Slack, MS Teams, VS Code, etc.). It just doesn't make sense to put time into developing a native desktop app that brings absolutely nothing over a browser-based client...
